pkgdown/_pkgdown.yml

```diff
@@ -57,7 +57,8 @@ reference:
       - logical
       - is_messydate
       - extract
-  - title: "Data"
-    desc: "Some test and/or example data:"
+  - title: "Reporting Data"
+    desc: "Function to help report on 'mdate' and example data:"
     contents:
+      - mreport
```

Comment:
Do we need a link to the article here somewhere, or not yet?

Reply:
I would wait a bit for now and, if we do add the article in the website in the future, we may need to change the output style. Though I am not sure we do need it since we already have a vignette and the README that should be helpful to users.